spec/models/chat_spec.rb

```diff
@@ -12,10 +12,44 @@
   describe '#users' do
     let(:user_a) { create(:user) }
     let(:user_b) { create(:user) }
-    let(:chat) { create(:chat, user_a: user_a, user_b: user_b) }
+    subject { create(:chat, user_a: user_a, user_b: user_b) }
 
     it 'should return chats collections' do
```

Comment:
Maybe the message of the `it` block isn't clear enough on what it is testing. It seems to be testing the `users` array, and the message says `should return chat collections.`